lua/smart-splits/resize-mode.lua

```diff
@@ -1,12 +1,18 @@
 local M = {}
 
 function M.start_resize_mode()
+  local config = require('smart-splits.config').config
   if vim.fn.mode() ~= 'n' then
     vim.notify('Resize mode must be triggered from normal mode', vim.log.levels.ERROR)
     return
   end
 
-  local quit_key = require('smart-splits.config').resize_mode_quit_key
+  local on_enter = config.resize_mode.hooks.on_enter
+  if type(on_enter) == 'function' then
+    on_enter()
+  end
```

Comment:
You should be able to just do `pcall(config.resize_mode.hooks.on_enter)` instead of this whole block